tests/integration/test_models/test_full_battery_models/test_lithium_ion/test_dfn.py

```diff
@@ -6,160 +6,12 @@
-        model = pybamm.lithium_ion.DFN(options)
-        var = pybamm.standard_spatial_vars
-        var_pts = {var.x_n: 10, var.x_s: 10, var.x_p: 10, var.r_n: 5, var.r_p: 5}
-        modeltest = tests.StandardModelTest(model, var_pts=var_pts)
-        modeltest.test_all()
-
-    def test_particle_uniform(self):
-        options = {"particle": "uniform profile"}
-        model = pybamm.lithium_ion.DFN(options)
-        modeltest = tests.StandardModelTest(model)
-        modeltest.test_all()
-
-    def test_particle_quadratic(self):
-        options = {"particle": "quadratic profile"}
-        model = pybamm.lithium_ion.DFN(options)
-        modeltest = tests.StandardModelTest(model)
-        modeltest.test_all()
-
-    def test_particle_quartic(self):
-        options = {"particle": "quartic profile"}
-        model = pybamm.lithium_ion.DFN(options)
-        modeltest = tests.StandardModelTest(model)
-        modeltest.test_all()
-
-    def test_loss_active_material_stress_negative(self):
-        options = {"loss of active material": ("stress-driven", "none")}
-        model = pybamm.lithium_ion.DFN(options)
-        chemistry = pybamm.parameter_sets.Ai2020
-        parameter_values = pybamm.ParameterValues(chemistry=chemistry)
-        modeltest = tests.StandardModelTest(model, parameter_values=parameter_values)
-        modeltest.test_all()
-
-    def test_loss_active_material_stress_positive(self):
-        options = {"loss of active material": ("none", "stress-driven")}
-        model = pybamm.lithium_ion.DFN(options)
-        chemistry = pybamm.parameter_sets.Ai2020
-        parameter_values = pybamm.ParameterValues(chemistry=chemistry)
-        modeltest = tests.StandardModelTest(model, parameter_values=parameter_values)
-        modeltest.test_all()
-
-    def test_loss_active_material_stress_both(self):
-        options = {"loss of active material": "stress-driven"}
-        model = pybamm.lithium_ion.DFN(options)
-        chemistry = pybamm.parameter_sets.Ai2020
-        parameter_values = pybamm.ParameterValues(chemistry=chemistry)
-        modeltest = tests.StandardModelTest(model, parameter_values=parameter_values)
-        modeltest.test_all()
-
-    def test_loss_active_material_reaction_both(self):
-        options = {"loss of active material": "reaction-driven"}
-        model = pybamm.lithium_ion.DFN(options)
-        modeltest = tests.StandardModelTest(model)
-        modeltest.test_all()
-
-    def test_surface_form_differential(self):
-        options = {"surface form": "differential"}
-        model = pybamm.lithium_ion.DFN(options)
-        modeltest = tests.StandardModelTest(model)
-        modeltest.test_all()
-
-    def test_surface_form_algebraic(self):
-        options = {"surface form": "algebraic"}
-        model = pybamm.lithium_ion.DFN(options)
-        modeltest = tests.StandardModelTest(model)
-        modeltest.test_all()
+class TestDFN(BaseIntegrationTestLithiumIon, unittest.TestCase):
+    def setUp(self):
```

Comment:
Shouldn't it be `setup` or `SetUp`?

Reply:
It needs to be `setUp`, it's a method of `unittest.TestCase` https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp